ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestClientBase.java

```diff
@@ -48,16 +55,16 @@ protected String generateEndpointSuggestion(String existingSuggestedEndpointUri,
                 existingEndpoint = new URIBuilder(dataSource);
                 endpointUriToSuggestStr = emendEndpointUri(existingEndpoint);
             } catch (URISyntaxException e) {
-                log.error("Couldn't parse dataSource '{}', so no suggestion can be made.", dataSource, e);
+                log.warn("Couldn't parse dataSource '{}', so no suggestion can be made.", dataSource, e);
```

Comment:
I think this message can be unclear to the user he didnt ask for any suggestion- lets say  : After failing the request - client tried to parse the URL and failed.

Reply:
Updated message